daisy_workflows/image_import/enterprise_linux/translate.py

```diff
@@ -87,16 +88,75 @@
 '''
 
 
+class NonFatalErrors:
```

Comment:
I find this class, with boolean fields instead of a list of errors a bit strange. I would have expected a list of errors instead. That way, each error type can have it's own formatting and the processing of potential causes can be done in a loop without the knowledge of details of each error (the IFs starting in line 359). With 3 different causes this is not that bad right now, but, if many more potential causes are added in the future, this piece of code and the return error IFs will become less pretty.

Reply:
Good point. In the current version, I switched to a list of matcherse.